docs/usage/self-hosted-configuration.md

```diff
@@ -213,6 +213,22 @@ For example:
 }
 ```
 
+## autodiscoverProjects
+
+You can use this option to filter the list of autodiscovered repositories by project names.
```

Comment:
I would like a extra line explaining _why_ this can be handy, and/or _what_ the intended use is.